src/lib/pay-request/services/webhooks/client.ts

```diff
@@ -11,7 +18,7 @@ export default class Webhooks extends Client {
   webhooks = ((client: Webhooks) => ({
     retrieve(
       id: string,
-      params: RetrieveWebhookRequest
+      params: RetrieveWebhookRequest | RetrieveWebhookRequestWithOverride
```

Comment:
Just for my own curiosity, can we just override the restriction by default? Is there any benefit to having this restriction from toolbox. Isn't it just there to prevent against broken access control in the admin tool/API?